internal/controller/promotions/promotions.go

```diff
@@ -665,3 +662,27 @@ func parseCreateActorAnnotation(promo *kargoapi.Promotion) string {
 	}
 	return creator
 }
+
+func calculateRequeueInterval(p *kargoapi.Promotion) time.Duration {
+	var (
```

Comment:
Nit: This seems unconventional. It's actually two extra lines of code to put all these vars in a `var` block vs just declaring each of them just-in-time with `:=` instead.